src/main/java/com/arcanewarrior/disguises/listeners/translation/MoveTranslation.java

```diff
@@ -0,0 +1,21 @@
+package com.arcanewarrior.disguises.listeners.translation;
+
+import com.arcanewarrior.disguises.Disguise;
+import com.arcanewarrior.disguises.DisguiseManager;
+import net.minestom.server.coordinate.Pos;
+import net.minestom.server.entity.Player;
+import net.minestom.server.event.player.PlayerMoveEvent;
+
+public final class MoveTranslation {
+    public static void listener(PlayerMoveEvent event, DisguiseManager parentManager) {
+        Player player = event.getPlayer();
+        Disguise disguise = parentManager.getPlayerDisguise(player);
+        if (disguise != null) {
```

Comment:
I'm not fond of this move translation, as teleporting the entity is a lot less smooth than translating the move packets from player to entity.

Reply:
Minestom use teleports in their move listener and it's fine. I tested it btw and it's smooth.